### PR TITLE
Feature/negative durations

### PR DESCRIPTION
**Pull request by DerHahne**
Hi, 
I implemented the feature request to enable negative time durations. The configuration is set to false by default. 

This implementation is regarding to feature request #3

### CODE REVIEW

File: src/app/duration-picker/duration-picker.component.html [hunk 1]
```diff
@@ -56,6 +59,14 @@
   </tr>
 
   <tr>
+    <td *ngIf="config.showNegative">
+      <input type="checkbox" id="dp-negative" [(ngModel)]="negative" #status [disabled]="disabled">
```
Comment (ShinDarth) [inline src/app/duration-picker/duration-picker.component.html]: why do we need this `<input>` ?

File: src/app/duration-picker/duration-picker.component.spec.ts [hunk 2]
```diff
@@ -59,11 +59,47 @@ describe('DurationPickerComponent', () => {
     expect(component.generate()).toBe('P9MT10M');
   });
 
+  it('generate() should correctly generate the negative duration values', () => {
```
Comment (ShinDarth) [inline src/app/duration-picker/duration-picker.component.spec.ts]: this test is failing for me

**Comment by ShinDarth** [comment]
@DerHahne  Thanks a lot for your contribution! Please check my review comments

File: src/app/duration-picker/duration-picker.component.spec.ts (hunk 2 above)
Comment (DerHahne) [reply src/app/duration-picker/duration-picker.component.spec.ts]: Fixed!

**Comment by DerHahne** [comment]
Hi!
I made some rework on input checkbox and the failing unit test. 

We have a binding on a variable by clicking the negative/positive state button without input checkbox. 

For the failing unit test i forgot to add a variable which must be set to true.

**Comment by ShinDarth** [comment]
@DerHahne merged, thanks a lot!

**Comment by DerHahne** [comment]
Thx @ShinDarth !!! :)

**Pull request merged**